js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,237 @@
+import React, { Fragment } from "react";
+import styled from "styled-components";
+import { Select } from "@yoast/components/src/select/Select";
+import { __, sprintf } from "@wordpress/i18n";
+import { createPortal } from "react-dom";
+import HelpIcon from "@yoast/components/src/help-icon/HelpIcon";
```

Comment:
We should not load in components directly like this, but expose them via the `index.js`.
```suggestion
import { HelpIcon } from "@yoast/components";
```

However, only the .css of the HelpIcon is exposed right now. So a change in the monorepo is needed for this.

Reply:
I have created (an PR)[https://github.com/Yoast/javascript/pull/756] which can (and should be) merged to address this problem :)